source/reference/minio-mc/mc-idp-ldap-accesskey-ls.rst

```diff
@@ -0,0 +1,189 @@
+.. _minio-mc-idp-ldap-accesskey-ls:
+
+============================
+``mc idp ldap accesskey ls``
+============================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+
+.. mc:: mc idp ldap accesskey list
+.. mc:: mc idp ldap accesskey ls
+
+
+Description
+-----------
+
+.. start-mc-idp-ldap-accesskey-ls-desc
+
+The :mc:`mc idp ldap accesskey ls` displays a list of LDAP access key pairs.
+
+.. end-mc-idp-ldap-accesskey-ls-desc
+
+:mc:`mc idp ldap accesskey ls` is also known as :mc:`mc idp ldap accesskey list`.
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+         The following example returns a list of access keys associated with the authenticated user on the ``minio`` :ref:`alias <alias>`:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc idp ldap accesskey ls minio/
+
+      If the authenticated user is the ``admin``, the example command returns a list of all users and their associated access keys.
```

Comment:
This implies that for users w/o `admin:*`  (or, I would venture, `admin:ListUsers` we can only return the accesskeys for the current user.

That should be up top before the examples.

Reply:
Another question for you @taran-p.
Can you confirm what permissions a user must have for the `admin` rule to take effect?

Reply:
Just `admin:ListUsers` for that to take effect, yes

Reply:
Thanks! Updating this page with that info.